src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -966,3 +966,124 @@
     text: >
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
+
+helps['acr scope-map'] = """
+type: group
+short-summary: Manage scope access maps for Azure Container Registries.
+"""
+
+helps['acr scope-map create'] = """
+type: command
+short-summary: Create a scope map for an Azure Container Registry.
+examples:
+  - name: Create a scope map that allows content/write and metadata/read actions for `hello-world` repository, and content/read action for `hello-world-again`.
+    text: >
+        az acr scope-map create -n MyScopeMap -r MyRegistry --add hello-world content/write metadata/read --add hello-world-again content/read --description "Sample scope map."
+"""
+
+helps['acr scope-map delete'] = """
+type: command
+short-summary: Delete a scope map under an Azure Container Registry.
+examples:
+  - name: Delete the scope map 'MyScopeMap'.
+    text: >
+        az acr scope-map delete -r MyRegistry -n MyScopeMap
+"""
+
+helps['acr scope-map update'] = """
+type: command
+short-summary: Update a scope map under an Azure Container Registry, appending/removing specified actions.
+examples:
+  - name: Update the scope map 'MyScopeMap' removing metadata/read and content/read actions for `hello-world` repository, and metadata/write action for `hello-world-again`.
+    text: >
+        az acr scope-map update -n MyScopeMap -r MyRegistry --remove hello-world metadata/read content/read --remove hello-world-again metadata/write
+"""
+
+helps['acr scope-map show'] = """
+type: command
+short-summary: Show details and attributes for a scope map under an Azure Container Registry.
+examples:
+  - name: Get information for the scope map 'MyScopeMap'.
+    text: >
+        az acr scope-map show -n MyScopeMap -r MyRegistry
+"""
+
+helps['acr scope-map list'] = """
+type: command
+short-summary: List all scope maps under an Azure Container Registry.
+examples:
+  - name: List scope maps under the registry 'MyRegistry'.
+    text: >
+        az acr scope-map list -r MyRegistry
+"""
+
+helps['acr token'] = """
+type: group
+short-summary: Manage tokens for an Azure Container Registry.
+"""
+
+helps['acr token create'] = """
+type: command
+short-summary: Create a token for an Azure Container Registry.
+examples:
+  - name: Create a token associated to the scope map 'MyScopeMap', in 'disabled' status.
```

Comment:
Create a token associated with the scope map 'MyScopeMap' in the 'disabled' status.